src/styles/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat($columnsAmount, 2);
```

Comment:
The value for `grid-template-columns` seems to be incorrect because the `repeat` function requires a unit for the second parameter (the size of the columns). You should specify a unit like `px`, `em`, `rem`, etc. For example: `repeat($columnsAmount, 200px);`.